woocommerce/woocommerce-android
# Enable coupons management by default

## Enable coupons management by default

Closes: #8993


### Description

1. Removed coupon management from beta features
2. Enabled by default in `MoreMenuViewModel`
3. Deleted unused strings and code

### Testing instructions

1. Verify coupon management is now available by default in the "More Menu" screen in the "General" section.
2. Verify coupon management is deleted from Settings > Beta features screen
